d3a_api_client/utils.py

```diff
@@ -239,12 +236,26 @@ def log_market_progression(message):
             headers.extend(["start_time", "duration_min", ])
             table_data.extend([data_dict.get("start_time"), data_dict.get("duration_min")])
 
-        logger.info(f"\n\n{tabulate([table_data, ], headers=headers, tablefmt='fancy_grid')}\n\n")
+        logging.info(f"\n\n{tabulate([table_data, ], headers=headers, tablefmt='fancy_grid')}\n\n")
     except Exception as e:
         logging.warning(f"Error while logging market progression {e}")
 
+
 domain_name_from_env = os.environ.get("API_CLIENT_DOMAIN_NAME", DEFAULT_DOMAIN_NAME)
 
 
 websocket_domain_name_from_env = os.environ.get("API_CLIENT_WEBSOCKET_DOMAIN_NAME", DEFAULT_WEBSOCKET_DOMAIN)
 
+
+def log_bid_offer_confirmation(message):
+    try:
+        if message.get("status") == "ready":
+            event = message.get("command")
+            data_dict = json.loads(message.get(event))
+            energy = data_dict.get("energy")
+            price = data_dict.get("price")
+            trader = data_dict.get("seller" if event=="offer" else "buyer")
+            logging.info(f"{trader} {'OFFERED' if event == 'offer' else 'BID'} "
+                         f"{round(energy, 2)} kWh at {price} cts/kWh")
+    except Exception as e:
+        logging.error(f"Error logging bid/offer info. Error details: {e}")
```

Comment:
I think this is a bit redundant: you would see "Error <something>" 3 times (one for the log level, and two for the printed messages). I think it would be better to refactor it somehow, maybe removing some part of the message.